int.ts

```diff
@@ -1786,7 +1808,16 @@ module J2ME {
                       // handles.
                       address = ref[--sp];
                       if (typeof address === "number") {
-                        args.unshift(getHandle(address));
+                        if (address === 0) {
+                          args.unshift(null);
+                        } else {
+                          var tmp = getArrayFromAddr(address);
+                          if (tmp) {
+                            args.unshift(tmp);
+                          } else {
+                            args.unshift(getHandle(address));
+                          }
```

Comment:
Nit: you could avoid the *tmp* with `args.unshift(getArrayFromAddr(address) || getHandle(address))`.